crates/dbs-boot/src/lib.rs

```diff
@@ -7,21 +7,40 @@
 //!
 //! This crate provides CPU architecture specific constants and utilities to abstract away CPU
 //! architecture specific details from the Dragonball Sandbox or other VMMs.
+use std::fmt;
 
 #[cfg(target_arch = "x86_64")]
 mod x86_64;
 #[cfg(target_arch = "x86_64")]
 pub use x86_64::*;
-
 #[cfg(target_arch = "aarch64")]
 mod aarch64;
 #[cfg(target_arch = "aarch64")]
 pub use aarch64::*;
 
+/// Types of devices that can get attached to this platform.
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Copy)]
+pub enum DeviceType {
```

Comment:
Seem it would be better to move this into dbs-arch?

Reply:
done